projects/isy-angular-widgets/src/lib/wizard/components/wizard/wizard.component.spec.ts

```diff
@@ -48,22 +47,21 @@ class TestComponent {
 let wizard: WizardComponent;
 let stepper: StepperComponent;
 let contentChildren: QueryList<WizardDirective>;
+let fixture: ComponentFixture<TestComponent>;
 
 describe('Integration Tests: WizardComponent with Mock Parent', () => {
-  let parentComponent: TestComponent;
-  let parentFixture: ComponentFixture<TestComponent>;
-
-  beforeEach(async () => {
-    await TestBed.configureTestingModule({
-      declarations: [TestComponent]
-    }).compileComponents();
+  let spectator: Spectator<TestComponent>;
+  const createdComponent = createComponentFactory({
+    component: TestComponent,
+    imports: [WizardModule, RouterTestingModule],
+    declarations: [MockComponent(IncompleteDateComponent)]
```

Comment:
Suggestion: No Mocks in integration test (like above)

Reply:
Done